modules/ROOT/pages/proc_installing_cli.adoc

```diff
@@ -0,0 +1,49 @@
+= Installing the CLI
+
```

Comment:
Did you use the templates? We need the id= anchor for this and the reference module below. Also keep in mind to base the file name and the ID on the module title.

Reply:
https://github.com/openshift/pipelines-docs/pull/3/commits/d5a43bcce34d400e8d85d18636490cdd54b46e25